m4_sessions/1806-inning/group7.md

```diff
@@ -1,33 +1,38 @@
 ## Group Members:
+Jim Dechant, Tony Robertson, Tyler Lundgren, Tyler McNamara
 
 
-
-## Name of Session: 
+## Name of Session: Lets just fuckin relax!
```

Comment:
I'd prefer a different title.  While some people are okay with swearing, it isn't for everyone and you don't want to exclude people right away